### ISSUE TITLE
Display "title text" of images onscreen

### ISSUE THREAD

**Issue by twm** [twm/yarrharr#87]
Not being able to see the title text is annoying when viewing webcomics, particularly xkcd.

**Comment by twm** [comment]
Note that this can't be done in pure CSS. Aside from the possible accessibility problems, using a rule like `img::after { content: attr(title); }` doesn't work because `<img>` is a replaced element.
